homeassistant/components/zha/backup.py

```diff
@@ -0,0 +1,21 @@
+"""Backup platform for the ZHA integration."""
+import logging
+
+from homeassistant.core import HomeAssistant
+
+from .core import ZHAGateway
+from .core.const import DATA_ZHA, DATA_ZHA_GATEWAY
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_pre_backup(hass: HomeAssistant) -> None:
```

Comment:
I assume these are from the HA backup platform? Seems odd to me to do the actual backup in something called `pre backup`

Reply:
ok I think I get the terminology... this is to perform our tasks before HA takes the backup.